http-client/src/test/java/org/triplea/http/client/throttle/size/MessageSizeThrottleTest.java

```diff
@@ -22,12 +22,11 @@ void messageSizeOverLimitIsNotAllowed() {
             .accept(ERROR_REPORT));
   }
 
-  @SuppressWarnings("JUnitTestMethodWithNoAssertions")
```

Comment:
That's a nice side effect

Reply:
It's just a theory at this point.  I'm assuming IntelliJ will actually recognize `assertDoesNotThrow()` as an assertion. :smile:  If you can verify when you get a chance, it would be appreciated.

Reply:
At least my IntelliJ didn't say anything so far